packages/framework/src/Model/Seo/SeoSettingFacade.php

```diff
@@ -9,34 +9,29 @@ class SeoSettingFacade
     public const SEO_TITLE_MAIN_PAGE = 'seoTitleMainPage';
     public const SEO_TITLE_ADD_ON = 'seoTitleAddOn';
     public const SEO_META_DESCRIPTION_MAIN_PAGE = 'seoMetaDescriptionMainPage';
-
-    /**
-     * @var \Shopsys\FrameworkBundle\Component\Setting\Setting
-     */
-    protected $setting;
+    public const SEO_ROBOTS_TXT_CONTENT = 'seoRobotsTxtContent';
 
     /**
      * @param \Shopsys\FrameworkBundle\Component\Setting\Setting $setting
      */
-    public function __construct(Setting $setting)
+    public function __construct(protected readonly Setting $setting)
     {
-        $this->setting = $setting;
     }
 
     /**
      * @param int $domainId
-     * @return string
+     * @return string|null
      */
-    public function getTitleMainPage($domainId)
+    public function getTitleMainPage(int $domainId): ?string
     {
         return $this->setting->getForDomain(self::SEO_TITLE_MAIN_PAGE, $domainId);
     }
 
     /**
      * @param int $domainId
-     * @return string
+     * @return string|null
      */
-    public function getDescriptionMainPage($domainId)
+    public function getDescriptionMainPage(int $domainId): ?string
```

Comment:
adding return type and typehint is a BC-break. please remove it.

see https://symfony.com/doc/current/contributing/code/bc.html

Reply:
Removed.